docs/components/forms.rst

```diff
@@ -268,6 +268,52 @@ To change the order of fields on your Form:
 #. Click the field you would like to move
 #. Drag the field to a new location
 
+.. vale off
+
+Progressive profiling
+*********************
+.. vale on
+
+Progressive profiling is a powerful feature used to reduce the length of Forms by hiding all the fields that are already known. This prevents your Contacts from feeling overwhelmed by massive Forms and even reduces the time it takes to fill out a Form if fields are already known to your Mautic instance and thus hidden for the Contact.
+
+Configuration
+==============
+
+There are two ways to configure a Form Field to only display when the asked values are unknown.
+
+First, choose the Form that you want to use for progressive profiling. Go to the Form Fields and open the field configuration of the field you want to use for progressive profiling. Change to the Behavior tab. Here, you can configure the behavior of the fields.
+
+.. note::
+  It's always recommended to use the Email field, even if it's already known because Mautic uses the Email as a unique identifier for Contacts. Additionally, ensure the submit button field is always visible. Otherwise, the Contact can't submit the Form.
+
+1. **Show when value exists**:
+if set to 'No,' Mautic checks whether the value for this field exists in the database or was provided in a former Form submission. If a value is found, Mautic won't display the field in the Form. If set to 'Yes,' Mautic displays the field regardless of whether a value is found or not. The default configuration for this option is 'Yes'.
```

Comment:
```suggestion
if set to 'No,' Mautic checks whether the value for this field exists in the database or if a previous Form submission provided it. If found, Mautic won't display the field in the Form. If set to 'Yes,' Mautic displays the field regardless of the existence of a value in the field. The default configuration for this option is 'Yes'.
```

This was a bit of a tricky one to convert into active tense! 😅

Reply:
Yeah it was! Thank you!